src/handlers/records-read.ts

```diff
@@ -86,4 +82,21 @@ export class RecordsReadHandler implements MethodHandler {
     };
     return messageReply;
   };
+
+  /**
+   * Creates a filter using `recordId` in given descriptor, if not given, `protocol` & `protocolPath` are used to create the filter instead.
+   * @param descriptor message descriptor with optional properties `recordId`, `protocol` and `protocolPath`
+   *
+   * @returns {Filter} with a Records interface as well as the appropriate filter params
+   */
+  public static createFilter(descriptor: RecordsReadDescriptor): Filter {
```

Comment:
Why `public`? If it's just for testing, I'd prefer to make this private, since testing this separately feels like testing implementation rather than behavior.

Reply:
It originally lived on RecordsRead interface and was used in multiple places before the Grants change, since it now lives within the handler and is only used here this could be private.

Reply:
Word, makes sense. Can you make it private?